packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -66,6 +67,12 @@ export const useCheckoutPage = props => {
 
     const checkoutStep = checkoutData && checkoutData.cart.checkoutStep;
 
+    const toggleActiveContent = useCallback(() => {
+        const nextContentState =
+            activeContent === 'checkout' ? 'addressBook' : 'checkout';
```

Comment:
Is there any other way to handle this? Once we have Auth Payment Information, we will have to add another ternary operator to support this.

Reply:
My use case was just the two states so the toggle made sense. This change makes sense in the scope of Auth Payment if that will be another "tab", but I'm not familiar with that scope. There's been very little direction period about this new tab pattern, so best we get a full design before putting too much work into it.